privacy-contracts/src/main/solidity/DefaultOnChainPrivacyGroupManagementContract.sol

```diff
@@ -12,54 +13,58 @@ contract DefaultOnChainPrivacyGroupManagementContract is OnChainPrivacyGroupMana
         return _version;
     }
 
-    // overrides
     function canExecute() external view returns (bool) {
         return _canExecute;
     }
 
     function lock() public {
         require(_canExecute);
+        require(tx.origin == _owner, "Origin not the owner.");
         _canExecute = false;
     }
 
     function unlock() public {
         require(!_canExecute);
+        require(tx.origin == _owner, "Origin not the owner.");
         _canExecute = true;
     }
 
-    function addParticipants(bytes32 _enclaveKey, bytes32[] memory _accounts) public returns (bool) {
+    function addParticipants(bytes32[] memory _accounts) public returns (bool) {
```

Comment:
I know that this was already here, but I don't like the fact we are calling the enclaveKey `accounts`.
Given we are changing the contract, can we fix this (for all methods)?

Reply:
Good plan!

Reply:
Done